### PR TITLE
Contact Form: field descriptions

### PR DESCRIPTION
**Pull request by braamgenis**
To resolve GH-69. Field for adding descriptions containing HTML below contact form fields.

### CODE REVIEW

File: widgets/contact/styles/default.less [hunk 1]
```diff
@@ -84,6 +84,14 @@
 			}
 		}
 	}
+
+	.sow-form-field-description {
+		font-size: 0.9em;
+		margin-top: 0.2em;
+		margin-left: 0.2em;
+		color: #999;
```
Comment (gregpriday) [inline widgets/contact/styles/default.less]: We should probably also have a field in the design section to change this default colour.

Comment (braamgenis) [inline widgets/contact/styles/default.less]: Okily dokily. :+1:

**Comment by braamgenis** [comment]
@gregpriday Added field to set description color. You think we should allow more customization? (Like font family, size, etc)

**Comment by gregpriday** [comment]
Font size would be a good one, but I wouldn't be keen on font family, or much more than that. Maybe an italics option. Italic vs non-italic would be a fairly common change.

**Pull request merged**